packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -2717,13 +2713,13 @@ void main() {
       );
 
       owner.performAction(expectedNodeId, SemanticsAction.copy);
-      verify(controls.handleCopy(any, any)).called(1);
+      expect(controls.didCopy, true);
```

Comment:
This change is making the test slightly less precise (the `.called(1)`) is missing. instead of just having a `didCopy` boolean, can we have a `copyCount` integer?

Reply:
Done